Calculator/Calculator/LinkedList.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  LinkedList.swift
+//  Calculator
+//
+//  Created by Whales on 2023/06/03.
+//
+
+class Node<T> {
+    var data: T?
+    var next: Node?
+    
+    init(data: T? = nil, next: Node? = nil) {
+        self.data = data
+        self.next = next
+    }
+}
+
+struct LinkedList<T> {
+    private var head: Node<T>?
+    private var tail: Node<T>?
+    private(set) var count: Int = 0
+    
+    var isEmpty: Bool {
+        head == nil
+    }
+    
+    func readHeadData() -> T? {
+        return head?.data
+    }
+    
+    func readTailData() -> T? {
+        return tail?.data
+    }
+    
+    mutating func append(data: T?) {
+        if isEmpty {
+            head = Node(data: data)
+            tail = head
+            count += 1
+            return
+        }
+        
+        let newNode = Node(data: data)
+        tail?.next = newNode
+        tail = newNode
+        count += 1
```

Comment:
사바사가 강한 부분이긴 하지만, 저는 if문을 분기점으로 사용할 경우에는 코드의 들여쓰기 계층을 동일하게 해주는 것을 선호하는 편입니다. 
그리고 이렇게 작성하면 count 변수에 +1해주는 동작은 따로 뺄 수 있겠죠?
참고만 하세요~ ㅎㅎ
```suggestion
        let newNode = Node(data: data)
        
        if isEmpty {
            head = newNode
            tail = head
        } else {
            tail?.next = newNode
            tail = newNode
        }
        
        count += 1
```

Reply:
오 깔끔하고 좋은 것 같아요!!!!! 감사합니다!!😃